navi.plugin.fish

```diff
@@ -1,21 +1,17 @@
-function navi-widget -d 'Call navi'
-  set -q NAVI_USE_FZF_ALL_INPUTS; or set -l NAVI_USE_FZF_ALL_INPUTS "true"
-  begin
-    navi --print | while read -l r; set result $result $r; end
-
-    if [ -n "$result" ]
-      echo $result
-
-      # Remove last token from commandline.
-      commandline -t ""
+function navi_key_bindings
+  function navi-widget -d "Show cheat sheet"
+    set -q FZF_TMUX_HEIGHT; or set FZF_TMUX_HEIGHT 40%
+    begin
+      set -lx FZF_DEFAULT_OPTS "--height $FZF_TMUX_HEIGHT $FZF_DEFAULT_OPTS --tiebreak=index --bind=ctrl-r:toggle-sort $FZF_CTRL_R_OPTS +m"
```

Comment:
✂️ (Actually, is this necessary? Why not let the user decide the defaults given his/her `.fishrc`?)

Reply:
It is not necessary. I basically left it from the fzf script without thinking deeply if it's necessary. Personally I never used the shortcut to toggle the shorting mode. In the fzf case the idea was probably that you can different options for different applications of fzf. Now the FZF_TMUX_HEIGHT variable doesn't seem to have an effect anyway. I'm also not sure if navi should support this or not.